public/app/features/dashboard-scene/serialization/transformSaveModelToScene.ts

```diff
@@ -333,7 +333,11 @@ export function buildGridItemForPanel(panel: PanelModel): DashboardGridItem {
 
   if (panel.libraryPanel) {
     vizPanelState.$behaviors!.push(
-      new LibraryPanelBehavior({ uid: panel.libraryPanel.uid, name: panel.libraryPanel.name })
+      new LibraryPanelBehavior({
+        uid: panel.libraryPanel.uid,
+        name: panel.libraryPanel.name,
+        title: panel.title,
```

Comment:
Hmmmm.... How can one set this title? Just tried to do that and only way i can see it being possible is editing the JSON directly.